src/app/modules/group/pages/group-access/group-access.component.html

```diff
@@ -16,8 +16,10 @@
       <ng-container *ngIf="rootActivityState.isReady">
         <ng-container *ngIf="rootActivityState.data; else noActivity">
           <div class="item">
-            <i class="icon fa fa-folder"></i>
-            <strong i18n>Root activity: {{ rootActivityState.data.string.title }}</strong>
+            <div>
+              <i class="icon fa fa-folder"></i>
+              <strong i18n>Root activity: {{ rootActivityState.data.string.title }}</strong>
+            </div>
```

Comment:
What is that new div for ?

Reply:
Group access page is common and has common styles. New section with permissions requires flex rules, so I had to add div here for avoid flex effect.